crates/i2c/src/main.rs

```diff
@@ -249,6 +249,9 @@ fn main() {
 
 #[cfg(test)]
 mod tests {
+
```

Comment:
Did rustfmt do this? There was no newline before

Reply:
rustfmt did yes